conf/carbon.properties.template

```diff
@@ -17,29 +17,25 @@
 #
 
 #################### System Configuration ##################
-#Mandatory. Carbon Store path
-carbon.storelocation=hdfs://hacluster/Opt/CarbonStore
+#Optional. Carbon Store path
```

Comment:
Mention that if it is not specified it takes spark warehouse path

Reply:
added